Makefile

```diff
@@ -7,15 +8,14 @@ CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false,maxDescLen=
 
 CONTROLLER_GEN_VERSION = v0.5.0
 GOLANGCI_VERSION = v1.33.0
-KUBEBUILDER_VERSION = 2.3.1
 LICENSEI_VERSION = v0.3.1
+ENVTEST_CTRL_VERSION = v0.8.3
```

Comment:
why not use latest version (0.9.0)?

Reply:
This example was pulled from a kubebuilder project with version v0.8.3, but kubebuilder itself hasn't been upgraded to v0.9.0 just yet. I expect there will be some other changes there as well, but this should be good enough for now.

Reply:
I mean there is better tooling for this in controller-runtime v0.9.0: https://github.com/kubernetes-sigs/controller-runtime/tree/v0.9.0/tools/setup-envtest

Reply:
I saw this, I just didn't want to put more effort into this for now (oh dear laziness). But since we will use the same for the other projects we should use this I have to agree...

Reply:
Laziness is indeed a good reason not to do this 😄 but then we should create some issues as a reminder that this needs to be updated sometime in the future. Also, a comment marking this as a legacy solution would be nice so that others don't try to copy this as is if they can help it.